app-doc/sphinx-basic-ng/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=sphinx-basic-ng
+PKGSEC=python
+PKGDEP="python-3"
+BUILDDEP="setuptools"
+PKGDES="A modern skeleton for Sphinx themes"
```

Comment:
Skeleton (what?) of Sphinx themes.

Please avoid advertising on behalf of the package, per the Styling Manual.